include/natalie/array_value.hpp

```diff
@@ -110,7 +113,8 @@ class ArrayValue : public Value {
     ValuePtr max(Env *);
     ValuePtr min(Env *);
     ValuePtr push(Env *, size_t, ValuePtr *);
-    ValuePtr ref(Env *, ValuePtr, ValuePtr);
+    ValuePtr rassoc(Env *, ValuePtr);
+    ValuePtr ref(Env *, ValuePtr, ValuePtr = nullptr);
```

Comment:
Giving the second argument a default value make it easy to use as a binding
But is that what we want? Or would a method `at` which just calls `ref` be better?

Reply:
I think I prefer the latter (method `at` which calls `ref`), just for clarity of purpose. :smile:

Reply:
Made it an explicit method